### ISSUE TITLE
Parse Candidate Error

### ISSUE THREAD

**Issue by shumy** [legastero/jingle-interop-demos#13]
On testing the strophe.jingle with OpenFire server gives me an error:
"parseCandidate called with a line that is not a candidate line"
"failed to get ice && jcandJingleSession.sendIceCandidate @ strophejingle.bundle.js:1399peerconnection.onicecandidate @ strophejingle.bundle.js:1281"

**Comment by fippo** [comment]
do you know the candidate.candidate argument passed to candidateToJingle? What browser are you testing with?

**Comment by fippo** [comment]
ah... i had fixed that issue a while ago in a1ff8fca9ba57741f43393452cb002c81ae6c934 but never pushed the commit. Let me know if the problem persists!

**Comment by shumy** [comment]
Yes, the error is gone, but still no connection! Chrome console output:

sendIceCandidate using trickle
strophejingle.bundle.js:6945 sending single candidate
strophejingle.bundle.js:7000 sendIceCandidates [RTCIceCandidate]
strophejingle.bundle.js:7033 try to send ack(transportinfo)...
strophejingle.bundle.js:6930 sendIceCandidate using trickle
strophejingle.bundle.js:6945 sending single candidate
strophejingle.bundle.js:7000 sendIceCandidates [RTCIceCandidate]
strophejingle.bundle.js:7033 try to send ack(transportinfo)...
strophejingle.bundle.js:6930 sendIceCandidate using trickle
strophejingle.bundle.js:6945 sending single candidate
strophejingle.bundle.js:7000 sendIceCandidates [RTCIceCandidate]
strophejingle.bundle.js:7033 try to send ack(transportinfo)...
strophejingle.bundle.js:6930 sendIceCandidate using trickle
strophejingle.bundle.js:6945 sending single candidate
strophejingle.bundle.js:7000 sendIceCandidates [RTCIceCandidate]
strophejingle.bundle.js:7033 try to send ack(transportinfo)...
strophejingle.bundle.js:6930 sendIceCandidate using trickle
strophejingle.bundle.js:6945 sending single candidate
strophejingle.bundle.js:7000 sendIceCandidates [RTCIceCandidate]
strophejingle.bundle.js:7033 try to send ack(transportinfo)...
strophejingle.bundle.js:6930 sendIceCandidate using trickle
strophejingle.bundle.js:6945 sending single candidate
strophejingle.bundle.js:7000 sendIceCandidates [RTCIceCandidate]
strophejingle.bundle.js:7033 try to send ack(transportinfo)...
strophejingle.bundle.js:6930 sendIceCandidate using trickle
strophejingle.bundle.js:6945 sending single candidate
strophejingle.bundle.js:7000 sendIceCandidates [RTCIceCandidate]
strophejingle.bundle.js:7033 try to send ack(transportinfo)...
strophejingle.bundle.js:6930 sendIceCandidate using trickle
strophejingle.bundle.js:6945 sending single candidate
strophejingle.bundle.js:7000 sendIceCandidates [RTCIceCandidate]
strophejingle.bundle.js:7033 try to send ack(transportinfo)...
strophejingle.bundle.js:6951 sendIceCandidate: last candidate...
strophejingle.bundle.js:6989 Have we encountered any srflx candidates? false
strophejingle.bundle.js:6990 Have we encountered any relay candidates? false
strophejingle.bundle.js:6993 no candidates found!
